src/Components/Common/Sidebar/SidebarUserCard.tsx

```diff
@@ -22,10 +22,10 @@ const SidebarUserCard = ({ shrinked }: { shrinked: boolean }) => {
         <i className="text-white text-3xl uil uil-user-circle" />
       </Link>
       <div className={`${shrinked ? "hidden" : "grow"} pl-3 flex flex-col`}>
-        <div className="h-6 flex items-center">
+        <div className="min-h-6 flex items-center">
           <Link
             href="/user/profile"
-            className="font-semibold text-white flex-nowrap overflow-hidden truncate"
+            className="font-semibold text-white flex-nowrap overflow-hidden break-all"
```

Comment:
How about breaking words instead?
```suggestion
            className="font-semibold text-white flex-nowrap overflow-hidden break-words"
```

Reply:
Fails in some cases where the name contains a single word that is too long to fit.

`break-words`
![image](https://user-images.githubusercontent.com/3626859/201298384-145000f3-e091-4abd-9eb7-9e76aad7fedb.png)

`break-all`
![image](https://user-images.githubusercontent.com/3626859/201298395-01ac8e0f-a567-4b4c-b389-0ce3a2fbc725.png)

(Fun fact: It's a real name!)

Reply:
the width is probably not correct then. it should ideally work if the width is correct. (https://tailwindcss.com/docs/word-break#break-words)
![image](https://user-images.githubusercontent.com/25143503/201299099-e245fb90-983c-4dae-90ba-117690656a6e.png)

Reply:
maybe that's why even `truncate` didn't work earlier

Reply:
Let me take a look at this, by the way what are the benefits of doing this over `break-all`?

Reply:
Got it working with `break-words`, Reference: https://codepen.io/lehollandaisvolant/pen/MvOYxp
![image](https://user-images.githubusercontent.com/3626859/201308302-0d2113e1-0b29-4075-abb9-569813cd6ca2.png)